.travis.yml

```diff
@@ -11,12 +12,20 @@ os:
 matrix:
   allow_failures:
     - node_js: 5
```

Comment:
Don't think we need to allow Node 5 to fail, considering we don't let it fail in https://github.com/meanjs/mean/blob/master/.travis.yml anymore